odoo/addons/base/models/ir_actions.py

```diff
@@ -682,18 +682,19 @@ def _get_warning_messages(self):
 
         if self.state == 'webhook' and self.model_id:
```

Comment:
Same as just this change?
```suggestion
        if self.state == 'webhook' and self.model_id.model:
```